media/css/firefox/family/components/_agreement.scss

```diff
@@ -6,30 +6,59 @@
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
 .c-agreement {
-    background-color: f3.$violet-dark;
+    background-color: $color-violet-60; /* darker bg needed for accessible color contrast */
```

Comment:
I only made the change here instead of updating the `$violet-dark` var because that var is used in the hero section as well, and our meta picture, so it seems best to leave the F3 var as is for now.